scripts/format.py

```diff
@@ -7,6 +7,7 @@
     out = open(outfile, "w")
     for l in open(f):
         if l.strip().startswith("{{") or l.strip().startswith("{%"):
+            # pylint: disable=consider-using-enumerate
```

Comment:
I didn't want to change the logic in this PR, so I simply disabled this check to pass the CI check.